### ISSUE TITLE
How to choose between tabs and spaces?

### ISSUE THREAD

**Issue by Simon-Will** [MJochim/webdav-editor#1]
I am editing a yaml file of the following form:

```
---
pageId: home
events:
    conference1:
        date: 2020-02-02
---
{% include content.md %}
```

All indentation consists of spaces. Now I want to enter a key-value pair `url: https://conf1.example.com/` in the `conference1` mapping, i.e. next to the `date` key. So I put my cursor after `2020-02-02` and hit Enter. Then four (not two, not eight) tabs are inserted. This is a little strange and may be a bug.

How can I fix this?

Alternatively, how can I display spaces and tabs in the editor, so I can at least discern tabs and spaces visually?

**Comment by MJochim** [comment]
I agree this might be a bug. None of this is currently configurable at runtime.

You can find the init code of the editors here:

https://github.com/MJochim/webdav-editor/blob/975bbffc269c7dc78e2cb387bb0d6e8c5a3cd489/src/views/file-editor-view.class.js#L160-L165

And I am guessing that your are referring to the Markdown editor. The init options it accepts are documented at https://github.com/sparksuite/simplemde-markdown-editor (right up front in `README.md`). You might find something suitable there. The project seems to be no longer maintained.

**Comment by Simon-Will** [comment]
I’m actually not sure what editor I refer to. I will check it tomorrow and report back.

**Comment by MJochim** [comment]
Only `*.md` are files are opened in the markdown editor (SimpleMDE). Everything else is opened with Monaco, which is “the code editor which powers VS Code.”

**Comment by Simon-Will** [comment]
Alright, then it must have been the markdown editor since the file was called `index.md`
